annotation-utils/src/main/java/edu/isi/vista/annotationutils/SerializedCasConverter.java

```diff
@@ -0,0 +1,111 @@
+package edu.isi.vista.annotationutils;
+
+import static org.apache.uima.cas.impl.Serialization.deserializeCASComplete;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.OutputStream;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.util.Map;
+import org.apache.uima.UIMAException;
+import org.apache.uima.cas.CAS;
+import org.apache.uima.cas.CASException;
+import org.apache.uima.cas.impl.CASCompleteSerializer;
+import org.apache.uima.cas.impl.CASImpl;
+import org.apache.uima.fit.util.JCasUtil;
+import org.apache.uima.jcas.JCas;
+import org.apache.uima.resource.metadata.TypeSystemDescription;
+import org.apache.uima.util.CasCreationUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import webanno.custom.CTEventSpan;
+import webanno.custom.CTEventSpanType;
+
+public class SerializedCasConverter {
+  private static final Logger log = LoggerFactory.getLogger(SerializedCasConverter.class);
+
+  public static void main(String[] args) {
+    // arg[0]: path of the binary CAS files (file extension .ser)
+    // arg[1]: path of the output file
+    if (args.length < 2) {
+      log.error("Expecting two arguments: path to the binary CAS file and path of the output file");
+      System.exit(1);
+    }
+
+    File input = new File(args[0]);
```

Comment:
Use `Parameters` rather than command-line arguments

Reply:
Switch to translate all the annotations in one go

Reply:
Using parameters file and handle multiple annotations in one run. Please check.